### PR TITLE
fix: #347 Loses values when create entity using callout generate error or warning.

### LINKED ISSUE #347: [Bug Report] Row values are lost when a new record is added
## Bug report
If at the moment of creating a record in the table of child tabs a change is made in a field containing a callout or an error is generated, the values are lost when updating the list of records.

#### Steps to reproduce
1. Open a window with child tabs, in this case 'Unit of Measure'.
2. Select the action from the 'New Record' table.
3. In the 'Multiply Rate' field set a value greater than zero.

#### Screenshot or Gif
Peek 19-02-2020 13-09

#### Expected behavior
If an error occurs or a warning is generated by an activated callout you must keep the data placed in the row you are editing.

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactor
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, not the `master` branch
- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your `master` branch!
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number): fix #347 

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

## Description

If when creating a record in the table of child tabs a change is made to a field containing a callout or an error or warning is generated due to a callout, it maintains the values entered by the user.

### Screenshots

 **Before this PR**
Peek 19-02-2020 13-09

**After this PR**
Peek 19-02-2020 14-34

### CODE REVIEW

File: src/utils/ADempiere/dictionaryUtils.js [hunk 1]
```diff
@@ -446,6 +446,9 @@ export function assignedGroup(fieldList, assignedGroup) {
   if (fieldList === undefined || fieldList.length <= 0) {
     return fieldList
   }
+
+  // TODO: Change to destructuring and add isParent attribure to change orderBy
+  // sequence value to seqNoGrid value if isParent is false
```
Comment (yamelsenih) [inline src/utils/ADempiere/dictionaryUtils.js]: Please remove this, if You need remember some just create a issue for it

File: src/utils/ADempiere/dictionaryUtils.js [hunk 2]
```diff
@@ -496,6 +499,8 @@ export function assignedGroup(fieldList, assignedGroup) {
  * @param {string} orderBy
  * @param {string} type
  * @param {string} panelType
+ * TODO: Change to destructuring and add isParent attribure to change orderBy
```
Comment (yamelsenih) [inline src/utils/ADempiere/dictionaryUtils.js]: Please remove this, if You need remember some just create a issue for it

**Review by yamelsenih — CHANGES_REQUESTED** [review]
Please remove this, if You need remember some just create a issue for it

The changes must be atomic of  functionality of bugfix

**Comment by EdwinBetanc0urt** [comment]
> Please remove this, if You need remember some just create a issue for it
> 
> The changes must be atomic of functionality of bugfix

Hi @yamelsenih the comments are deleted. Is add issue to it's comments.

**Pull request merged**